great_expectations/checkpoint/actions.py

```diff
@@ -352,7 +352,6 @@ def _send_slack_notification(self, payload: dict) -> dict:
         return {"slack_notification_result": slack_notif_result}
 
 
-@public_api
```

Comment:
Are these actions no longer GX support or no longer community supported either?

That is, do we want community supported actions in our public docs and labeled as such or not appearing at all.

If these aren't community supported, can we delete the actions altogether?

Reply:
> Are these actions no longer GX support or no longer community supported either?

These actions are unsupported by GX _and_ unsupported by the community (source of truth for integration support is https://docs.greatexpectations.io/docs/application_integration_support#integrations) 

> If these aren't community supported, can we delete the actions altogether?

Yes, but that's outside the scope of this docs PR. I'll file a followup issue for this and product/eng can then prioritize as y'all see fit (I'm not sure how impactful the tech debt is / if it's worth your time to clean it up or easier to just leave it in place in case product decides we should support these integrations in the future)

Reply:
We should always remove dead code.

Reply:
Thinking about this more, maybe we need to leave the code in for semantic versioning. Removing the public api decorator will necessitate a minor version bump.